#########################################################################
# ⭐⭐⭐  please enter your waypoints   ⭐⭐⭐ #
#########################################################################

drone_001_waypoints:
  - {x: -2.0, y: 2.5, z: 1.5}
  - {x: -1.5, y: 2.5, z: 1.5}
  - {x: -1.0, y: 2.5, z: 1.5}
  - {x: -0.5, y: 2.5, z: 1.5}
  - {x: -0.0, y: 2.5, z: 1.5}
  - {x: 0.5, y: 2.5, z: 1.5}
  - {x: 2.0, y: 2.5, z: 1.5}
  - {x: 2.5, y: 2.5, z: 1.5}
  - {x: 3.0, y: 2.5, z: 1.5}
  # ... 其他航点 ...

drone_002_waypoints:
  - {x: -2.0, y: -0.5, z: 1.5}
  - {x: -1.5, y: -0.5, z: 1.5}
  - {x: -1.0, y: -0.5, z: 1.5}
  - {x: -0.5, y: -0.5, z: 1.5}
  - {x: -0.0, y: -0.5, z: 1.5}
  - {x: 0.5, y: -0.5, z: 1.5}
  - {x: 2.0, y: -0.5, z: 1.5}
  - {x: 2.5, y: -0.5, z: 1.5}
  - {x: 3.0, y: -0.5, z: 1.5}
  # ... 其他航点 ...

drone_003_waypoints:
  - {x: -3.0, y: 1.01, z: 1.5}
  - {x: -2.5, y: 1.01, z: 1.5}
  - {x: -2.0, y: 1.01, z: 1.5}
  - {x: -1.5, y: 1.01, z: 1.5}
  - {x: -1.0, y: 1.01, z: 1.5}
  - {x: -0.5, y: 1.01, z: 1.5}
  - {x: 1.0, y: 1.01, z: 1.5}
  - {x: 1.5, y: 1.01, z: 1.5}
  - {x: 2.0, y: 1.01, z: 1.5}
  # ... 其他航点 ...

drone_004_waypoints:
  - {x: -1.0, y: -4.0, z: 1.5}
  - {x: -1.0, y: -4.25, z: 1.5}
  - {x: -1.0, y: -4.5, z: 1.5}
  - {x: -1.0, y: -4.75, z: 1.5}
  - {x: -1.0, y: -5.0, z: 1.5}
  - {x: -1.0, y: -5.25, z: 1.5}
  - {x: -1.0, y: -5.5, z: 1.5}
  - {x: -1.0, y: -5.75, z: 1.5}
  # ... 其他航点 ...

drone_005_waypoints:
  - {x: -1.0, y: -4.0, z: 1.5}
  - {x: -1.0, y: -4.25, z: 1.5}
  - {x: -1.0, y: -4.5, z: 1.5}
  - {x: -1.0, y: -4.75, z: 1.5}
  - {x: -1.0, y: -5.0, z: 1.5}
  - {x: -1.0, y: -5.25, z: 1.5}
  - {x: -1.0, y: -5.5, z: 1.5}
  - {x: -1.0, y: -5.75, z: 1.5}
  # ... 其他航点 ...

threshold : 0.1
#左后x:-5 y 3.2
#左前x:8.6 y 3.2
#右前x 7.2 y-2.7
#我 x-5.2 y-2.7
